src/inputs/labeled-inputs.scss

```diff
@@ -229,6 +230,29 @@
     border-top-left-radius: 0;
     border-bottom-left-radius: 0;
   }
+
+  &.iui-actionable {
+    align-items: center;
+    height: calc(100% - 2px);
```

Comment:
The `-2px` may need to change if a status line is applied on the bottom:
![Screen Shot 2021-10-01 at 2 09 23 PM](https://user-images.githubusercontent.com/849817/135667397-fb4de529-5665-4425-970b-2101a681fc1b.png)

Reply:
I'm reverting height to 90%. @bentleyvk